bagel/tests/test_cli_bids.py

```diff
@@ -151,3 +156,39 @@ def test_create_acquisitions(bids_path, bids_dir, acquisitions, bids_session):
 
     for contrast, count in acquisitions.items():
         assert image_counts[contrast] == count
+
+
+@pytest.mark.parametrize(
+    "bids_sub_id, session",
+    [("01", "01"), ("02", "02"), ("03", "01")],
+)
+def test_get_session_path_session_exists(bids_sub_id, session):
```

Comment:
either in the test name or docstring, I'd explicitly state that this is the test for the case where the session layer **does** exist. so either `test_get_session_path_when_session_exists` (bit long) or just add it to the docstring.